app/src/main/res/layout/main_fragment.xml

```diff
@@ -11,7 +11,7 @@
         android:id="@+id/actStock"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:hint="Enter stock"
+        android:hint="@string/placeholder"
```

Comment:
I went ahead and made a placeholder for the "enter stock" on the text line.
This will allow you to call the value multiple;e times, rather than enter it manually each time a new text line is made.

Reply:
String externalization is good for internationalization.

Give it a name that is more descriptive.  placeholder is ambiguous.